presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/StatisticsBuilders.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata.statistics;
+
+import com.facebook.presto.orc.ColumnWriterOptions;
+import com.facebook.presto.orc.metadata.OrcType;
+
+import java.util.function.Supplier;
+
+public class StatisticsBuilders
```

Comment:
A new class providing a factory method creating a supplier for flat map key stats builder.